crowdsourcing/viewsets/task.py

```diff
@@ -261,3 +284,37 @@ class CurrencyViewSet(viewsets.ModelViewSet):
 
     queryset = Currency.objects.all()
     serializer_class = CurrencySerializer
+
+
+class ExternalSubmit(APIView):
+    def post(self, request, *args, **kwargs):
+        identifier = request.GET.get('daemo_id', False)
```

Comment:
is that a case when it's a multi step task or something? the results field can be anything list/object so Yes in multiple requests we store them as a list, if we know it's a re-submit then we update the existing one, this is independent of whether it's an Ajax call or regular form POST (there can be multiple forms)